src/pk/asn1/der/utf8/der_length_utf8_string.c

```diff
@@ -48,10 +48,10 @@ unsigned long der_utf8_charsize(const wchar_t c)
 int der_utf8_valid_char(const wchar_t c)
 {
    LTC_UNUSED_PARAM(c);
-#if !defined(__WCHAR_MAX__) || __WCHAR_MAX__ > 0xFFFF
+#if !defined(LTC_WCHAR_MAX) || LTC_WCHAR_MAX > 0xFFFF
    if (c > 0x10FFFF) return 0;
 #endif
-#if !defined(__WCHAR_MAX__) || __WCHAR_MAX__ != 0xFFFF && __WCHAR_MAX__ != 0xFFFFFFFF
+#if !defined(LTC_WCHAR_MAX) || LTC_WCHAR_MAX != 0xFFFF && LTC_WCHAR_MAX != 0xFFFFFFFF
```

Comment:
> is this !defined(... part still required?

it is because:
```
#if defined(__WCHAR_MAX__)
#define LTC_WCHAR_MAX __WCHAR_MAX__
#elif defined(WCHAR_MAX)
#define LTC_WCHAR_MAX WCHAR_MAX
#else
/* !!!!!!! here LTC_WCHAR_MAX remains undefined */
#endif
```